imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/PluginManifest.java

```diff
@@ -0,0 +1,55 @@
+package pl.edu.uj.kimage.plugin;
+
+public class PluginManifest {
```

Comment:
Are you sure this is the right name? [Wiki](https://en.wikipedia.org/wiki/Manifest_file)